packages/sol-apy-sdk/index.ts

```diff
@@ -100,5 +106,10 @@ export const calcAverageApy = (
     return null;
   }
 
-  return priceChange ** (millisecondsInAYear / deltaMilliseconds) - 1;
+  return {
+    apy: priceChange ** (millisecondsInAYear / deltaMilliseconds) - 1,
+    epochs: priceRecords.length < epochs ? priceRecords.length : epochs,
```

Comment:
please change this to epochEnd - epochStart, it is not guaranteed that the number of records matches number of epochs